transcriptic/cli.py

```diff
@@ -484,6 +484,46 @@ def launch_cmd(ctx, protocol, project, save_input, local, accept_quote, params,
     api = ctx.obj.api
     commands.launch(api, protocol, project, save_input, local, accept_quote, params, pm=None, test=None, pkg=None)
 
+@cli.command('preview-params')
+@click.argument('protocol')
+@click.option(
+    '--project', '-p',
+    metavar='PROJECT_ID',
+    required=True,
+    help='Project id or name context for configuring the protocol. Use '
+         '`transcriptic projects` command to list existing projects.'
+)
+@click.option(
+    '--local',
+    is_flag=True,
+    required=False,
+    help='If include, the specified protocol will launch a from the local manifest file.'
+)
+@click.option(
```

Comment:
it seems like the default behavior currently is to generate preview params to a file, with the name controlled by this flag.

If not specified, I think the ideal behavior here would be to write to stdout